Starting to support Jakarta EE 11 M1

arjantijms opened the pull request:
This adds support for several Jakarta EE 11 M1 APIs. There is a LOT of patching being used, mostly because of https://github.com/eclipse-ee4j/jersey/issues/5492 and https://github.com/eclipse-ee4j/parsson/issues/118

Expressly needs to be compiled locally from the Expressly master, since a the moment the CI doesn't work. See https://gitlab.eclipse.org/eclipsefdn/helpdesk/-/issues/4056

The Jakarta Security TCK did pass, except for the signature check which needs to be regenerated. Many other tests and likely TCKs will fail, as this is a very early work in progress.

kaido207 committed:
Remove 'sort="true"' set in 'filterLabels' in 'targetResourceTableButtons.inc'

Signed-off-by: kaido207 <kaido.hiroki@fujitsu.com>

kaido207 committed:
Disable the delete button if no instance is selected in New Cluster Page

Signed-off-by: kaido207 <kaido.hiroki@fujitsu.com>

kaido207 committed:
Fix When opening any Cluster's Resource page, all items in menu tree are highlighted

Signed-off-by: kaido207 <kaido.hiroki@fujitsu.com>

kaido207 committed:
Add a display area for error messages when file upload fails

Signed-off-by: kaido207 <kaido.hiroki@fujitsu.com>

kaido207 committed:
Fix the symbol for the required field is not displayed with Location

Signed-off-by: kaido207 <kaido.hiroki@fujitsu.com>

kaido207 committed:
Formatting

Signed-off-by: kaido207 <kaido.hiroki@fujitsu.com>

kaido207 committed:
Fix Fail to deploy a app when the app's material files remain and if there is a lifecycle module

Signed-off-by: kaido207 <kaido.hiroki@fujitsu.com>

kaido207 committed:
Add TLS checkbox in AdminGUI

Signed-off-by: kaido207 <kaido.hiroki@fujitsu.com>

arjantijms committed:
Fix WebSocket not working for default web module applications.

Signed-off-by: Arjan Tijms <arjan.tijms@omnifish.ee>

arjantijms committed:
Processing feedback

* Removing redundant semi-collon
* Moving dummy registration to separate class

Signed-off-by: Arjan Tijms <arjan.tijms@omnifish.ee>

arjantijms committed:
Removing semi-colon

Signed-off-by: Arjan Tijms <arjan.tijms@omnifish.ee>

kaido207 committed:
Refactor near fixes

Signed-off-by: kaido207 <kaido.hiroki@fujitsu.com>

arjantijms committed:
Merge pull request #24691 from arjantijms/websocket_default_webmodule

Fix WebSocket not working for default web module applications.

arjantijms committed:
Merge pull request #24692 from kaido207/minor_fixes_admingui

Minor Fixes and Improvements in AdminGUI

arjantijms committed:
Merge pull request #24694 from kaido207/uncheck_lifecycle_location

Fix #24693 Fail to deploy a app when the app's material files remain and if there is a lifecycle module

arjantijms committed:
Integrate Jersey 3.1.4

Signed-off-by: Arjan Tijms <arjan.tijms@omnifish.ee>

arjantijms committed:
Merge pull request #24695 from arjantijms/jersey_314

Integrate Jersey 3.1.4

arjantijms committed:
Fix accidentally changed constant

Signed-off-by: Arjan Tijms <arjan.tijms@omnifish.ee>

arjantijms committed:
Merge pull request #24697 from arjantijms/fix_autologin_adminui

Fix auto login admin ui by accidentally changed constant

arjantijms committed:
Merge remote-tracking branch 'origin/master' into 8.0

dmatej committed:
Fixed several possible issues with rolling log files

- Failed rollToNewFile could cause NPE
- ATOMIC_MOVE is not supported on all file systems (some Windows FS)
- Cleanup should be executed just if roll succeeded

Signed-off-by: David Matějček <david.matejcek@omnifish.ee>

glassfish-bot committed:
Prepare release 7.0.11

dmatej committed:
Switched to System.Logger and renameTo replaced by Files.move

Signed-off-by: David Matějček <david.matejcek@omnifish.ee>

glassfish-bot committed:
Prepare next development cycle for 7.0.12-SNAPSHOT

arjantijms committed:
Merge pull request #24701 from eclipse-ee4j/7.0.11-BRANCH

dmatej committed:
Merge pull request #24700 from dmatej/fix-logging

Fixed several possible issues with rolling log files

zistrong committed:
fix cmtTimeoutInSeconds config, it can't take effect when set value 0 in admin UI.

arjantijms committed:
Merge pull request #24703 from kaido207/add_gui_tls11_13

Add TLS checkbox in AdminGUI

arjantijms committed:
Webpage update for 7.0.11

Signed-off-by: Arjan Tijms <arjan.tijms@omnifish.ee>

arjantijms committed:
Add initial support for several Jakarta EE 11 M1 APIs

Signed-off-by: Arjan Tijms <arjan.tijms@omnifish.ee>

hs536 reviewed (approved):
In the first stages of adapting to the new Jakarta EE, we should allow for a certain amount of brokenness.

hs536 committed:
Merge pull request #24704 from arjantijms/update_website7011

Webpage update for 7.0.11

dmatej committed:
Merge pull request #24702 from zistrong/7.0.11-bugfix

fix cmtTimeoutInSeconds config, it can't take effect when set value 0 in admin UI.

arjantijms committed:
Added Mojarra 4.1 M1 (as snapshot) support.

Faces TCK runs to some degree. It (obviously) fails on the Jakarta
Validation support, which has not yet been updated.

Signed-off-by: Arjan Tijms <arjan.tijms@omnifish.ee>

arjantijms committed:
Expressly finally as M1 instead of SNAPSHOT

Signed-off-by: Arjan Tijms <arjan.tijms@omnifish.ee>

arjantijms committed:
Added M1 versions of Faces and Mojarra

Signed-off-by: Arjan Tijms <arjan.tijms@omnifish.ee>

arjantijms committed:
Merge remote-tracking branch 'origin/master' into 8.0

arjantijms committed:
Integrate Jersey 3.1.5

Signed-off-by: Arjan Tijms <arjan.tijms@omnifish.ee>

arjantijms committed:
Merge pull request #24706 from arjantijms/jersey_315

Integrate Jersey 3.1.5

arjantijms committed:
Add Jakarta Validation 3.1.0-M1

Add patch for Hibernate-validator and some code to easier find OSGi
version mismatches

Signed-off-by: Arjan Tijms <arjan.tijms@omnifish.ee>

arjantijms committed:
Merge remote-tracking branch 'origin/master' into 8.0

arjantijms committed:
Next version is 8.0.0

Signed-off-by: Arjan Tijms <arjan.tijms@omnifish.ee>

arjantijms committed:
Working on getting more tests to pass.

Disabling some tests for now, which of course should be enabled later
again.

Signed-off-by: Arjan Tijms <arjan.tijms@omnifish.ee>

arjantijms committed:
Use M1 for jsf templating

Signed-off-by: Arjan Tijms <arjan.tijms@omnifish.ee>

arjantijms committed:
Fix several tests

Signed-off-by: Arjan Tijms <arjan.tijms@omnifish.ee>

arjantijms committed:
Use M1 for Exousia again

Signed-off-by: Arjan Tijms <arjan.tijms@omnifish.ee>

arjantijms commented:
Build, mvn tests and many application tests pass now. Not everything works of course, but this is a very early access PR.
